### ISSUE TITLE
0x80370102 troubleshooting documentation is insufficient

### ISSUE THREAD

**Issue by HunterZ** [MicrosoftDocs/WSL#1395]
Encountered the following error when trying to launch Ubuntu via WSL2:
```
Installing, this may take a few minutes...
WslRegisterDistribution failed with error: 0x80370102
Error: 0x80370102 The virtual machine could not be started because a required feature is not installed.

Press any key to continue...
```

Consulted the following page for help:
https://docs.microsoft.com/en-us/windows/wsl/troubleshooting

Information here is insufficient for solving the issue. Systeminfo, coreinfo, and task manager all show full virtualization support, and VirtualBox works fine:
```
Hyper-V Requirements:      VM Monitor Mode Extensions: Yes
                           Virtualization Enabled In Firmware: Yes
                           Second Level Address Translation: Yes
                           Data Execution Prevention Available: Yes
```

"Virtual Machine Platform" and "Windows Subsystem for Linux" Windows Features are enabled, which are reportedly all that is needed for WSL2.

I'm running Windows 11 version 21H2 22000.376.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: d83a849a-3167-a76c-7a10-caa1eb11b025
* Version Independent ID: 91d93e7d-8206-77bf-ace0-f8c44810bf17
* Content: [Troubleshooting Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/troubleshooting)
* Content Source: [WSL/troubleshooting.md](https://github.com/MicrosoftDocs/wsl/blob/main/WSL/troubleshooting.md)
* Product: **dev-environment**
* Technology: **windows-subsystem-for-linux**
* GitHub Login: @craigloewen-msft
* Microsoft Alias: **crloewen**

**Comment by craigloewen-msft** [comment]
Can you please file this as an issue on the [Microsoft/WSL repo](https://github.com/microsoft/wsl) instead? Filling out the template there will give us the necessary info to help you, and this repo is just for the docs pages of WSL. 

Can you also run `(gcim Win32_ComputerSystem).HypervisorPresent` in PowerShell and show the output? (This will check if the hypervisor is actually present). ANd can you ensure that you have the "Windows Subsystem for Linux" optional component enabled by running: `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart` in PowerShell? 

Thanks for filing this!

**Comment by HunterZ** [comment]
No, I'm not going to do this. There's already a bug on this that was opened and subsequently ignored, despite more and more people showing up to report that they're also encountering the issue: https://github.com/microsoft/WSL/issues/7068

The documentation is insufficient, or else we would be able to resolve this without having to go to the WSL Github for extra help.

**Comment by numeralz** [comment]
If anyone else is still struggling with this, and are tired of seeing "Enable virtualization in BIOS" because it's enabled:

For me, there was a `hunter1.sys` driver, which I believe was installed as part of an anti-cheat mechanism from some Steam game I had installed recently.

How I discovered it:  
`Windows Security > Device Security > Memory Integrity > ON`

Gives me some error of a conflicting driver: 
hunter1.sys

I found the `hunter1.sys`, deleted it, rebooted. BINGO. WSL 2 now works as if nothing had ever broke -.-

Hope this helps someone avoid reinstalling Windows to fix.
